lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -22,19 +22,19 @@ blockchain_configuration:
   block0_consensus: bft
 
   # Number of slots in each epoch
-  slots_per_epoch: 2160
+  slots_per_epoch: 500
```

Comment:
Just to avoid _too many_ blocks in the schedule which makes the logger panics otherwise :man_shrugging: ... And, it'll be fun to not use 2160 slots per epoch, I am sure of it.